### PR TITLE
Connecting controller collage_photo with UCW home page

### PR DESCRIPTION
**Pull request by KamilWo**
Modified template of collage_photo module.
Added dependency in bestja_configuration_ucw.

### CODE REVIEW

**Comment by KrzysiekJ** [comment]
This pull request has conflict(s).

**Comment by KamilWo** [comment]
Reason for the conflicts remains unknown. Rebasing complete.

File: addons/bestja_frontend_ucw/views/home_page.xml [hunk 1]
```diff
@@ -302,107 +301,51 @@
...
-                                            }
-                                            } );
-                                        </script>
+                                        </t>
                                     </div>
+                                    <script src="/bestja_frontend_ucw/static/src/js/modernizr.min.js"></script>
+                                    <script src="/bestja_frontend_ucw/static/src/js/classie.js"></script>
+                                    <script src="/bestja_frontend_ucw/static/src/js/photostack.js"></script>
+                                    <script>
+                                        // [].slice.call( document.querySelectorAll( '.photostack' ) ).forEach( function( el ) { new Photostack( el ); } );
```
Comment (KrzysiekJ) [inline addons/bestja_frontend_ucw/views/home_page.xml]: The commented out code should be either removed or there should be a comment explaining why it is present.

File: addons/bestja_frontend_ucw/views/home_page.xml [hunk 2]
```diff
@@ -302,107 +301,51 @@
...
+
+                                        new Photostack( document.getElementById( 'photostack-1' ), {
+                                        callback : function( item ) {
```
Comment (KrzysiekJ) [inline addons/bestja_frontend_ucw/views/home_page.xml]: Wrong indentation.

File: addons/bestja_frontend_ucw/views/home_page.xml (hunk 1 above)
Comment (KamilWo) [inline addons/bestja_frontend_ucw/views/home_page.xml]: It has been removed, long time ago commented and unnecessary now. Photostack is working without this code.

File: addons/bestja_frontend_ucw/views/home_page.xml (hunk 2 above)
Comment (KamilWo) [inline addons/bestja_frontend_ucw/views/home_page.xml]: Its true, all clear now, thank you.

**Comment by KrzysiekJ** [comment]
Merged in 690c882.

**Pull request closed without merging**